package.json

```diff
@@ -1,4 +1,16 @@
 {
+  "name": "stac-extensions",
+  "version": "1.0.0",
+  "dependencies": {
```

Comment:
these should be all `devDependencies` as they are not required if you `npm install stac-extensions` (granted im assuming we should not be publishing this package.

Reply:
I've applied this suggestion to https://github.com/linz/STAC/pull/9.

Reply:
I've applied this suggestion to https://github.com/linz/STAC/pull/9.